tools/functional-tester/etcd-tester/main.go

```diff
@@ -49,6 +51,8 @@ func main() {
 	consistencyCheck := flag.Bool("consistency-check", true, "true to check consistency (revision, hash)")
 	isV2Only := flag.Bool("v2-only", false, "'true' to run V2 only tester.")
 	stresserType := flag.String("stresser", "default", "specify stresser (\"default\" or \"nop\").")
+	failureTypes := flag.String("failures", "default,failpoints", "specify failures (concat of \"default\" and \"failpoints\").")
+	terminate := flag.Bool("terminate", true, "'true' to terminate an etcd cluster after testing.")
```

Comment:
I see. Then I'll update this PR for only decoupling failures. And the idea of `--failure-wrapper` seems to be good for namazu. I'll create another PR for it later.